src/Components/ImagesTable/RegionsPopover.js

```diff
@@ -73,7 +75,24 @@ export const RegionsPopover = ({ composeId }) => {
     <Popover
       aria-label="Launch instance"
       headerContent={<div>Launch instance</div>}
-      bodyContent={<ul>{listItems}</ul>}
+      bodyContent={
+        <>
+          <ul>{listItems}</ul>
+          {!isBeta() && (
+            <>
+              <Divider className="pf-u-mt-sm pf-u-mb-sm" />
+              <Button
+                isInline
+                component="a"
+                variant="link"
+                href="https://console.redhat.com/beta/insights/image-builder/landing"
```

Comment:
This will kick us out of the development environment, or out of stage and into production... if we just use `href="/beta/insights/image-builder/landing` then whatever environment we are using should be preserved after clicking it.